LICENSE.txt

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2023, conda-forge
```

Comment:
This is the correct one and should stay.

Reply:
Done